release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1314,6 +1326,7 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Machine Config Operator*
 
+* Previously, rpm-ostree related operations were not handled properly on non-CoreOS nodes such as {op-system-first}. As a result, RHEL nodes were degraded when an operation, such as switching kernel, was applied in the pool that contained RHEL nodes. With this update, Machine Config Daemon logs a message whenever a non-supported operation is performed on non-CoreOS nodes. After logging the message, it returns nil instead of an error. RHEL nodes in the pool now proceed as expected when an unsupported operation is performed by Machine Config. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952368[*BZ#1952368*])
```

Comment:
s/RHEL/{op-system-base} in three places
s/switching kernel/kernel switching ?
s/Machine Config Daemon/the Machine Config Daemon
s/by Machine Config/by the Machine Config Daemon ?